src/components/TokenDetailAction.js

```diff
@@ -0,0 +1,438 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import $ from 'jquery';
+import ModalPin from '../components/ModalPin';
+import ReactLoading from 'react-loading';
+import hathorLib from '@hathor/wallet-lib';
+
+
+class TokenMint extends React.Component {
```

Comment:
Add description of what the component does. In this case, it is used to mint new tokens, right?

Reply:
Done